runner/runners/bazel.go

```diff
@@ -103,6 +117,7 @@ func postProcessBazel(filer snapshot.Filer,
 			StdoutDigest:      stdoutDigest,
 			StderrDigest:      stderrDigest,
 		},
+		ActionDigest: cmd.ExecuteRequest.ActionDigest,
```

Comment:
Do we need to check if cmd.ExecuteRequest is nil to avoid a nil pointer dereference?

Reply:
We should never hit that case, the first thing we do in the bazel workflow is check that ExecReq is non nil

Reply:
Gotcha. Think it'd still be worth using the helper method cmd.ExecuteRequest.GetActionDigest() from https://github.com/twitter/scoot/pull/362/files#diff-9776fc4ea2a6e05f737bf7c9ef6106bfR58?

Reply:
ohh, yes, thanks